### PR TITLE
Preferences dialog now closes on CLOSE DIALOG key binding

### LINKED ISSUE #8888: ESC should close the dialog
### JabRef version

Latest development branch build (please note build date below)

### Operating system

Windows

### Details on version and operating system

_No response_

### Checked with the latest development build

- [X] I made a backup of my libraries before testing the latest development version.
- [X] I have tested the latest development version and the problem persists

### Steps to reproduce the behaviour

1. Open preferences
2. Select "Citation key generator"\
   grafik
3. Press <kbd>ESC</kbd>
4. nothing happens
5. Select line "Article" in the table
6. Press <kbd>ESC</kbd>
7. Nothing happens

Expectation: at step 4 and 7, the dialog should close

### Appendix

...
<details>

<summary>Log File</summary>

```
Paste an excerpt of your log file here
```

</details>

### PR DESCRIPTION
**Pull request by sjb-ch1mp**
Fixes #8888 

The Preferences dialog was not closing when the CLOSE DIALOG key binding was pressed. This was because when the user changed the tab, they needed to click the Preference Tab List. This ListView was not respecting the event handlers that were present on the dialog. To resolve this, an 'onKeyReleased' event was added to the ListView. The Preferences tab now closes when the CLOSE DIALOG key binding is pressed.

- [x] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [x] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [x] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

**Review by Siedlerchr — APPROVED** [review]
Thanks for your contribution! LGTM! Needs review from a second dev

**Review by k3KAW8Pnf7mkmdSMPHz27 — APPROVED** [review]
LGTM and thank you for your contribution @sjb-ch1mp! 

A minor point is that this only partially fixes https://github.com/JabRef/jabref/issues/8888, as it only addresses it for one of the `ListView`s so I'll re-open the issue after this PR is merged

**Pull request merged**
